admin_manual/configuration_server/occ_command.rst

```diff
@@ -487,6 +488,17 @@ This is example converts SQLite to MySQL/MariaDB::
 For a more detailed explanation see 
 :doc:`../configuration_database/db_conversion`
 
+.. _database_add_indices_label:
+
+Add missing indices
+-------------------
+
+It might happen that we add from time to time new indices to already existing database tables,
+for example to improve performance. In order to check your database for missing indices run
+following command:
+
+ sudo -u www-data php occ db:add-missing-indices
+
```

Comment:
Should we add a note here that this can take a significant amount of time?
And that it is recommended to properly put the system in maintenance mode etc etc.?